src/benchData.c

```diff
@@ -1021,10 +1021,19 @@ int prepare_sample_data(int db_index, int stb_index) {
     }
 
     if (stbInfo->iface == REST_IFACE || stbInfo->iface == SML_REST_IFACE) {
-        if (convertHostToServAddr(g_arguments->host, g_arguments->port,
-                                  &(g_arguments->serv_addr))) {
-            errorPrint("%s\n", "convert host to server address");
-            return -1;
+        if (stbInfo->tcpTransfer && stbInfo->iface == SML_REST_IFACE) {
+            if (convertHostToServAddr(g_arguments->host, 6046,
```

Comment:
how about using the number passed by cmd line option instead of hard code?

Reply:
> how about using the number passed by cmd line option instead of hard code?

telnet TCP port is not related with our 6030 port number, which is fixed as 6046, I will replace this magic number with pre-defined